src/templates/docs.js

```diff
@@ -132,7 +132,7 @@ export default class MDXRuntimeTest extends React.Component {
         <PageTitle>
           <TitleWrapper>
             <Title>{docTitle}</Title>
-            {docsLocation && ((editable && mdx.frontmatter.editable !== false) || mdx.frontmatter.editable === true) ? (
+            {!isMobile() && docsLocation && ((editable && mdx.frontmatter.editable !== false) || mdx.frontmatter.editable === true) ? (
```

Comment:
Could you please check this statement? If I understand it correctly, I think we can simplify it to.
```ts
!isMobile() && docsLocation && (editable || mdx.frontmatter.editable)
```
I know it is existing code but it is good if we can improve it.

Thanks.

Reply:
I think this should not edit because it is a JS not TS, using `===` for type safe.

Reply:
FYI @mildronize TypeScript's already handled this for us because this `==` can cause an issue.
https://stackoverflow.com/questions/57125700/why-use-triple-equal-in-typescript

I haven't realized this is JavaScript file not TS.
BTW how about this one. 
````js
!isMobile() === true && docsLocation === true && (editable === true || mdx.frontmatter.editable === true )
```

I think this statement looks strange `((editable && mdx.frontmatter.editable !== false) || mdx.frontmatter.editable === true) `

Reply:
Yes, Let's others improve this.